aries-site/src/pages/components/search.mdx

```diff
@@ -0,0 +1,91 @@
+import { Example } from '../../layouts';
+import {
+  SearchExample,
+  SearchIconPositionExample,
+  SearchSimpleExample,
+  SearchSuggestionsExample,
+} from '../../examples';
+
+<Example
+  code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/components/search/SearchExample.js"
+  designer="https://designer.grommet.io/search?id=HPE-design-system-hpedesignsystem-hpe-com&mode=edit"
+  docs="https://v2.grommet.io/textinput?theme=hpe#props"
+  figma="https://www.figma.com/file/KKHWJN4GAI0Mq5yh0snDT6/HPE-Search-Component?node-id=265%3A112"
+  width="medium"
+>
+  <SearchExample />
+</Example>
+
+## Guidance
+
+Search inputs provide a powerful means for users to access content quickly and efficiently.
+
+Many search best practices extend "behind the scenes," such as how query strings are matched to relevant content and to the efficiency in which that information is retrieved.
+
+### Auto-suggestions
+
+- Provide effective [auto-suggestions](#search-with-auto-suggestions) whenever possible.
```

Comment:
Is this working for you? The deep linking behavior doesn't seem to be working for me...

Reply:
Yes, it works for me. Jumps down to the proper spot on the page.